src/utils/googleFormUtils.js

```diff
@@ -0,0 +1,9 @@
+export const filledInFormUrl = (projectName, projectId, phoneType) => {
+    const baseUrl = 'https://docs.google.com/forms/d/e/1FAIpQLScq85a1bh9QRgSsKpUOfXB0B-o1y5llJx3AyWhxGyq--LtCrw/viewform?usp=pp_url&'
```

Comment:
Can you explain how this form works? It seems a bit odd to have this base url hardcoded.

Reply:
Explained in person. Basically we only have one form for all projects and we autofill some of the fields based on which project is being reviewed

Reply:
You could POST directly to a Google Sheet if the form and responses are all shared anyway and write your own form with hidden auto filled inputs.

Reply:
I just mean write your own web form with the inputs for the questions being asked and the `onSubmit` event handler would POST to the Google Sheets API directly rather than sending them over to a Google Form with auto-filled inputs that are vulnerable to editing.

Sheets API append method that takes a spreadsheet ID as a parameter: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/append

Hidden input types: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden

I've done a lot of Google API work in the past couple of years and we almost went this route for a Intro 2 Astro feature.

Reply:
Or the update method might be a better option to submit to a Google Sheet: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/update

I forgot to say, Google Forms, just submit to an associated Google Sheet, so if Google Forms doesn't do what we want it to be doing, we could write our own web form instead.

Reply:
Oh for sure. Yeah, ideally we just make a native form for the user to fill out, but the whole purpose of using google forms is to just have a quick solution that anyone on the team can go in and modify. I think Google forms gives us most of the important functionality with almost no work.